backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -78,9 +78,12 @@ fun Route.configureShorkInterpreterControllerV1() {
     }
 
     post("/lobby") {
-        val lobbyId = shorkUseCase.createLobby()
-        call.respond(HttpStatusCode.Created, lobbyId.toString())
-        return@post
+        val creatingPlayerName = call.receive<String>()
+        val lobbyId = shorkUseCase.createLobby(creatingPlayerName)
+        if (lobbyId < 0) {
+            return@post call.respond(HttpStatusCode.BadRequest, lobbyId.toString())
+        }
```

Comment:
Please use a Result here. This looks like a prime example for a [Domain violation](https://github.com/corewar-teamprojekt/corewar/wiki/Error-Handling)